message/job.py

```diff
@@ -0,0 +1,71 @@
+# ############################################################################### #
+# Autoreduction Repository : https://github.com/ISISScientificComputing/autoreduce
+#
+# Copyright &copy; 2020 ISIS Rutherford Appleton Laboratory UKRI
+# SPDX - License - Identifier: GPL-3.0-or-later
+# ############################################################################### #
+""" Represents the messages passed between AMQ queues"""
```

Comment:
~~""" Represents the messages passed between AMQ queues"""~~
"""
Represents the messages passed between AMQ queues
"""